packages/docs/components/page-configs/va-toast/api-options.ts

```diff
@@ -0,0 +1,23 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
In demo this doesn't work properly
![image](https://user-images.githubusercontent.com/5394573/103086993-38f8cc00-45f7-11eb-86f0-ca346f2c0ad4.png)

I'm not seeing how that could be useful for end user.
Instead can we return said id on notification create.

Reply:
I want to clarify this point.
I understanding it something like this: 
`
const toast = $vaToast.init('Hello')
toast // is unique id
`
Is it correct?

Reply:
@damirJa yeah. That would work.